Update prepare-for-upload-vhd-image.md

rossanov opened the pull request:
wrong OS version for the proposed KB: 6.2.9200.22168 - KB4022718 for Win 2012 R2 is obviously wrong. 
Cheking KB4022726 instead, we see that  win32k.sys will be udated to 6.3.9600.18698

azurecla commented:
Hi __@Kevin-M-Smith__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

rossanov committed:
Update prepare-for-upload-vhd-image.md

wrong OS version for the proposed KB: 6.2.9200.22168 - KB4022718 for Win 2012 R2 is obviously wrong. 
Cheking KB4022726 instead, we see that  win32k.sys will be udated to 6.3.9600.18698
